versions.cmake

```diff
@@ -0,0 +1,23 @@
+superbuild_set_revision(occt
+  URL     "https://github.com/Open-Cascade-SAS/OCCT/archive/refs/tags/V7_6_3.tar.gz"
+  URL_MD5 00f8dc72a7cf378598f52e2216a15ea4)
+
+superbuild_set_revision(assimp
+  URL     "https://github.com/assimp/assimp/archive/refs/tags/v5.2.5.tar.gz"
+  URL_MD5 0b5a5a2714f1126b9931cdb95f512c91)
+
+superbuild_set_revision(imath
+  URL     "https://github.com/AcademySoftwareFoundation/Imath/archive/refs/tags/v3.1.5.tar.gz"
+  URL_MD5 dd375574276c54872b7b3d54053baff0)
+
+superbuild_set_revision(alembic
+  URL     "https://github.com/alembic/alembic/archive/refs/tags/1.8.3.tar.gz"
+  URL_MD5 2cd8d6e5a3ac4a014e24a4b04f4fadf9)
+
+superbuild_set_revision(vtk
+  URL     "https://www.vtk.org/files/release/9.2/VTK-9.2.0.rc2.tar.gz"
```

Comment:
no source package available yet

Reply:
in any case, I will need to add a superbuild_source_selection, we want ot be able to build against different version of VTK and for the actual package, it will be a specific commit, not a release.

Reply:
If you need an archive for 9.2: https://github.com/Kitware/VTK/archive/refs/tags/v9.2.0.tar.gz

Reply:
This will need to be updated once the archive drop anyway.